server/community.go

```diff
@@ -64,3 +67,58 @@ func (s *Server) postPRWelcomeMessage(ctx context.Context, pr *model.PullRequest
 
 	return nil
 }
+
+func (s *Server) assignGreeter(ctx context.Context, pr *model.PullRequest, repo *Repository) error {
+	// Only assign an greeter for non-member PRs
+	if s.IsOrgMember(pr.Username) {
+		return nil
+	}
+
+	// Is it configured to have a greeting team to handle the PR?
+	if repo.GreetingTeam == "" {
+		return nil
+	}
+
+	// get available members
+	greeterList, _, err := s.GithubClient.Teams.ListTeamMembersBySlug(ctx, s.Config.Org, repo.GreetingTeam, nil)
+	if err != nil {
+		return errors.Wrapf(err, "Error retrieving team members from %s", repo.GreetingTeam)
```

Comment:
Nit: error messages should not be capitalized: https://github.com/golang/go/wiki/CodeReviewComments#error-strings. (Here and below too)